pkg/desktop/desktop.go

```diff
@@ -0,0 +1,160 @@
+package desktop
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/browser"
+	"github.com/stashapp/stash/pkg/database"
+	"github.com/stashapp/stash/pkg/logger"
+	"github.com/stashapp/stash/pkg/manager/config"
+	"github.com/stashapp/stash/pkg/utils"
+	"golang.org/x/term"
+)
+
+func Start() {
+	if IsDesktop() {
+		OpenURLInBrowser(false, "")
+		writeStashIcon()
+		startSystray()
+	}
+}
+
+// OpenURLInBrowser opens a browser to the Stash UI. Path can be an empty string for main page.
+func OpenURLInBrowser(force bool, path string) {
+	// This can be done before actually starting the server, as modern browsers will
+	// automatically reload the page if a local port is closed at page load and then opened.
+	c := config.GetInstance()
+	if force || (!c.GetNoBrowser() && IsDesktop()) {
```

Comment:
I think the `force` parameter should be removed and the other conditions should be moved into `Start` so that `OpenURLInBrowser` is only called under these conditions.

Reply:
Refactored to remove the force option and check before calling